executor/app.py

```
@@ -0,0 +1,504 @@
+        status, message, error_detail = await agent_service.close_all_agent_sessions()
+        if status == TaskStatus.SUCCESS:
+            return {"message": message}
+        else:
+            # Return 200 status code even with errors, as some agents may have closed successfully
+            return {
+                "message": message,
+                "partial_success": True,
+                "error_detail": error_detail,
+            }
+    except Exception as e:
+        logger.exception(f"Error closing agent connections: {str(e)}")
```

Ruff (0.14.14)

Use explicit conversion flag

Replace with conversion flag

(RUF010)